### PR TITLE
Database from diagram

### PR DESCRIPTION
**Pull request by MattGomb**
In this PR we created a new file as per the requirements of [Create a database based on a diagram](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md) and filled it with the code that is required to create the database based on the diagram in the exercise.

The diagram:
image

The results of the required code (the code itself is in the .SQL file of course): 
newclin1
The last table `treatments_histories` is the join-table for the patient histories and treatments tables (many-to-many connections)

newclin2

newclin3

newclin4

newclin5

newclin6

newclin7

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+ALTER TABLE invoice_items ADD CONSTRAINT fk_treatments FOREIGN KEY (treatment_id) REFERENCES treatments(id);
+
+CREATE TABLE treatments_histories(
+  medical_history_id INT,
+  treatment_id INT,
+  PRIMARY KEY(medical_history_id, treatment_id),
+  CONSTRAINT fk_medical_history FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+  CONSTRAINT fk_treatment FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
```
Comment (Ismailco) [inline schema_based_on_diagram.sql:53]: - [ ] Good work creating the tables, however kindly consider creating indexes for all FOREIGN KEYS

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @MattGomb, @sonyco-4u,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (MattGomb) [reply schema_based_on_diagram.sql:53]: @Ismailco Hi, thanks, but why do we need to create indexes..? It is not suggested anywhere that we have to.

Comment (Ismailco) [reply schema_based_on_diagram.sql:53]: @MattGomb, The last point on the requirement suggests that you should add indexes for FK (foreign keys)

Screenshot_20221220_220738

Comment (MattGomb) [reply schema_based_on_diagram.sql:53]: @Ismailco  damn, how did i not see that 🤣  thanks!

**Comment by MattGomb** [comment]
required changes applied:

image

**Review by codecaiine — APPROVED** [review]
Hi @MattGomb @sonyco-4u , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of gitflow :heavy_check_mark: 
- Good database implementation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**